### PR TITLE
reverse_int

### PR DESCRIPTION
**Pull request by MelindaFang-code**
reverse the integer
e.g. 123 -> 321

### CODE REVIEW

File: benchmarks/core/reverse.bril [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+@main{
+  input: int = const 123;
```
Comment (sampsyo) [inline benchmarks/core/reverse.bril:3]: This is OK, but to make the benchmark a little more reusable, you might consider making the input a parameter. Here's an example:
https://github.com/sampsyo/bril/blob/a7a0759b20e724260825fc9c6ae7273a09c0bafd/benchmarks/core/armstrong.bril#L1-L2

That is, you can make `input` a parameter to `@main` and then give it a value like `# ARGS: 123` for benchmark automation purposes.

**Review by sampsyo — COMMENTED** [review]
Super cool and compact idea! Here is one minor suggestion. And it would also be great to include #263 to this PR.

**Comment by sampsyo** [comment]
Hmm… it looks like this may have erroneously reformatted the whole `bench.md` file?

**Comment by MelindaFang-code** [comment]
ooops it seems that it autoformats when I save the file. I have pushed a fix. @sampsyo

**Comment by sampsyo** [comment]
Looks good; thanks!

**Pull request merged**